### DISCUSSION TITLE
Inline-Blocks are not inline when added through the "+" button

### DISCUSSION THREAD

**Discussion by ChannKan** [payloadcms/payload#9419] [General]
### Describe the Bug

When adding a inline-block through the side "**+**" button in a rich-text field it is added as a block under the current line and not inline.
Because this is the general behavior of the "**+**" button inline-blocks should not be enabled.     

### Link to the code that reproduces this issue

https://github.com/shefing/payload_Inline_block_with_relationship/tree/Inline_block_with_relationship

### Reproduction Steps

**Steps to reproduce:**

1. Clone the link to reproduction ripo and launch Payload
2. Create a new Post instance
3. Type text into the rich-text field
4. Add a inline-block in the same line as the text with the "**+**" button 

**Expected:**
Inline-block is added after the text in the same line

**Actual:**
Inline-block is added under the text in the next line (Attached screenshot)
![Image](https://github.com/user-attachments/assets/7b89a21c-ebe4-4966-b753-9a50f1f1089a)

### Which area(s) are affected? (Select all that apply)

plugin: richtext-lexical

### Environment Info

Output from 'pnpm payload info':
Binaries:
Node: 20.9.0
npm: 10.2.5
Yarn: 1.22.21
pnpm: 9.10.0
Relevant Packages:
payload: 3.0.0-beta.104
next: 15.0.0-canary.104
@payloadcms/db-mongodb: 3.0.0-beta.104
@payloadcms/graphql: 3.0.0-beta.104
@payloadcms/live-preview: 3.0.0-beta.104
@payloadcms/next/utilities: 3.0.0-beta.104
@payloadcms/plugin-cloud-storage: 3.0.0-beta.104
@payloadcms/plugin-redirects: 3.0.0-beta.104
@payloadcms/plugin-seo: 3.0.0-beta.104
@payloadcms/richtext-lexical: 3.0.0-beta.104
@payloadcms/storage-s3: 3.0.0-beta.104
@payloadcms/translations: 3.0.0-beta.104
@payloadcms/ui/shared: 3.0.0-beta.104
react: 19.0.0-rc-06d0b89e-20240801
react-dom: 19.0.0-rc-06d0b89e-20240801

**Comment by AlessioGr** [comment]
This is currently the expected behavior and not a bug. As it’s not considered a bug, I’ll move this to a discussion. I’m open to discussing what the UX should look like

My thoughts: the first thing the + button does is create a new paragraph, and only then does it open the menu - it’s not designed for operations on the same line. It’s also the most convenient way to add a new paragraph after a block node, thus I think we should keep this behavior.

If we removed it from the + button, the only way to create inline blocks would be the fixed toolbar, and not everyone has it enabled.

**Reply by GermanJablo** [reply]
Thanks for reporting this, @ChannKan. Building on Alessio's points, we've discussed this internally and recognize that it’s a decision with a degree of subjectivity.

We'll track the feedback we get on this. Since I know a lot of people don't want to comment to share their opinion, I'm going to make a poll with 3 options, please everyone who reads this, react to this comment with the emoji that matches your preference:

🚀 -> Keep inline blocks under the '+' menu and insert them **at the cursor position**.
👀 -> Keep inline blocks under the '+' menu and insert them **in the next paragraph**.
🎉 -> Don't show inline blocks under the '+' menu.

As a final note, remember that you can now insert inline blocks at the cursor position by typing the `/` command.
